internal/gitea/repo.go

```diff
@@ -0,0 +1,218 @@
+package gitea
```

Comment:
Pre-emptive note regarding the package comment: there's no need to repeat it in each file -- you should only have it in one file, otherwise they'll be concatenated together and duplicated.